pkg/server/plugin/datastore/sql/sql.go

```diff
@@ -1274,6 +1300,19 @@ func modelToEntry(tx *gorm.DB, model RegisteredEntry) (*common.RegistrationEntry
 		})
 	}
 
+	var fetchedDNSs []*DNS
+	if err := tx.Debug().Model(&model).Related(&fetchedDNSs).Error; err != nil {
```

Comment:
What does this call to `Debug()` do?

Reply:
forgot to get rid of when I was having sql issues :)